ethcore/src/miner/miner.rs

```diff
@@ -614,12 +615,19 @@ impl Miner {
 							}
 						}).unwrap_or(default_origin);
 
+						// try to install zgp-checker before appending transactions
```

Comment:
@gavofyork @tomusdrw I personally agree with @tomusdrw , but I think maybe you both could reach a consensus in this question :)